From a200118d059ec46d29ef2e5b71892d60d31a6103 Mon Sep 17 00:00:00 2001
From: Elliott Sales de Andrade <quantum.analyst@gmail.com>
Date: Sun, 28 Jun 2020 17:37:36 -0400
Subject: [PATCH 3/7] Increase default tolerance slightly for 32-bit systems.

Around 900 tests fail on 32-bit systems, but with rather small RMS.

Signed-off-by: Elliott Sales de Andrade <quantum.analyst@gmail.com>
---
 lib/matplotlib/testing/decorators.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lib/matplotlib/testing/decorators.py b/lib/matplotlib/testing/decorators.py
index cb6812d7e..207612d2a 100644
--- a/lib/matplotlib/testing/decorators.py
+++ b/lib/matplotlib/testing/decorators.py
@@ -378,6 +378,8 @@ def image_comparison(baseline_images, extensions=None, tol=0,
         extensions = ['png', 'pdf', 'svg']
     if savefig_kwarg is None:
         savefig_kwarg = dict()  # default no kwargs to savefig
+    if sys.maxsize <= 2**32:
+        tol += 0.06
     return _pytest_image_comparison(
         baseline_images=baseline_images, extensions=extensions, tol=tol,
         freetype_version=freetype_version, remove_text=remove_text,
-- 
2.25.4

